Assets/Scripts/GfxReplayPlayer.cs

```diff
@@ -32,30 +32,30 @@ public struct MovementData
 
     void Awake()
     {
-        _highlightManager = GetComponent<HighlightManager>();
+        _highlightManager = FindObjectOfType<HighlightManager>();
```

Comment:
See [this discussion](https://cvmlp.slack.com/archives/C05HWV5UZ1A/p1707154210737549?thread_ts=1707148506.826349&cid=C05HWV5UZ1A). Is there a good reason to have HighlightManager be attached to a separate object? Also it would be great for it to be dynamically created via code rather than manually created in the Editor.